history/models.py

```diff
@@ -513,7 +516,10 @@ def get_nn(self,train=True):
 
         if train:
             for i in range(self.epochs):
-                TRAINER.train()
+                self.training_error = TRAINER.train()
+
+        cv=CrossValidator(trainer=TRAINER, dataset=DS, n_folds=5) 
```

Comment:
reading a little more on this, i think DS needs to be on a different dataset than the system was trained on.